### ISSUE TITLE
"Export as" outputs the original reference name

### ISSUE THREAD

**Issue by edoardocavazza** [timocov/dts-bundle-generator#117]
**Bug report**

When using the `export context as namespace` statement, the bundled output declares the original reference name (`context`) instead of the exported one (`namespace`).

**Input code**

```ts
// lib.ts
let context: any = {};
export { context as namespace };

// main.ts
export * from './lib.ts';
```

**Expected output**

```ts
export declare namespace: any;
```

**Actual output**

```ts
export declare context: any;
```

**Additional context**

I am using default CLI options.

**Comment by edoardocavazza** [comment]
Probably related to #116

**Comment by timocov** [comment]
I think it could be partially fixed in the following way:

**Expected output**
```typescript
declare let context: any;
export { context as namespace }
```

or something like this. It won't be renaming itself, just handling re-exports with renaming from the entry point.

**Comment by timocov** [comment]
Just published in 5.0 release.
